frontend/src/metabase/public/components/widgets/LegaleseStep/LegaleseStep.tsx

```diff
@@ -0,0 +1,43 @@
+import { t } from "ttag";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { LegaleseStepDetailsContainer } from "metabase/public/components/widgets/LegaleseStep/LegaleseStep.styled";
+import { Text, Button, Center, Stack, Title } from "metabase/ui";
+
+export const LegaleseStep = ({
+  goToNextStep,
+}: {
+  goToNextStep: () => void;
+}) => (
+  <Center bg="white" px="18rem" pt="6.25rem" pb="11.75rem">
+    <Stack align="center" spacing="3rem">
+      <Title order={2} fz="1.25rem">{t`First, some legalese`}</Title>
+
+      <LegaleseStepDetailsContainer p="lg" w="40rem">
+        <Text fw={700}>
+          {t`By clicking “Agree and continue” you're agreeing to`}{" "}
```

Comment:
Will splitting this sentence mess up the translation? I only ask because splitting the string here, between `…to` and `our embedding license` might introduce some issues when translating to languages that depend on prepositions for cases. I might be misunderstanding how the translation works here but I've been getting uneasy about splitting strings up like this so just want to make sure.

The issue exists in Estonian but I'll put a Russian example since @deniskaber can more easily verify it better than I can. I also know it's google translate which might be very wrong, but I think it still can get the general idea across

<img width="809" alt="image" src="https://github.com/metabase/metabase/assets/25306947/42b38b7b-8558-40ed-9136-b456835ce786">

<img width="812" alt="image" src="https://github.com/metabase/metabase/assets/25306947/1abcfcf3-402d-44aa-8fc3-5b3db015f9bb">

(i.e. `нашей лицензией` vs `наша лицензия`)

Reply:
Aren't our translations manual/hand-written anyway?
As far as I understand it, the translator will just have 2 keys/"original sentences" to translate but they can be translated as we want.

I looked if using `jt` helps with that, I believe I found the issue where jt originated from: https://github.com/ttag-org/ttag/issues/3#issuecomment-285283686 (funny that the usecase case is also translating something with a link in it :) )
If what they wrote it's correct it seems like even with jt the full context will be missed (as it creates two "keys").

Reply:
Ok then let's leave it as it is, and let the translators handle it

Reply:
The difference would be what would be presented to the translators.

In this case, there would be 2 strings
1. "By clicking “Agree and continue” you're agreeing to"
2. "our embedding license."

if you use `jt` e.g. 
```js
jt`By click... ${(<ExternalLink ...>{t`our embedding license.`}</ExternalLink>)}`
```

the translators would see
1. "By clicking “Agree and continue” you're agreeing to {0}"
2. "our embedding license."

which might give them a bit more context that there is some more texts after the first sentence.

tl;dr I like the use of `jt` in this case.

Reply:
Ok I think the `jt` will work just fine here - it's weird that it still gives two keys (I mean, I understand why, but still) but if we can give the translators as much context then let's give it to them